apps/agreement/package.json

```diff
@@ -23,9 +23,7 @@
     "apm:publish:minor": "aragon apm publish minor --files public/ --prepublish-script apm:prepublish",
```

Comment:
Should we revert these changes, especially if we've rebased the base branches to be on top of `master`?